inference-engine/ie_bridges/java/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2020 Intel Corporation
+
+find_package(Java REQUIRED)
+include(UseJava)
+
+set(JAVA_AWT_INCLUDE_PATH NotNeeded)
+
+project(inference_engine_java_api)
+
+set(InferenceEngine_LIBRARIES inference_engine)
+
+find_package(JNI REQUIRED)
+
+file(GLOB_RECURSE sources cpp/*.cpp)
+add_library(${PROJECT_NAME} SHARED ${sources})
+
+file(GLOB_RECURSE java_source org/intel/openvino/*.java)
+add_jar(inference_engine_jar ${java_source} 
+        OUTPUT_NAME ${PROJECT_NAME} 
+        OUTPUT_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+
+target_link_libraries(${PROJECT_NAME} PRIVATE ${InferenceEngine_LIBRARIES})
```

Comment:
we can remove `InferenceEngine_LIBRARIES` variable and use `inference_engine` directly here